packages/terser/README.md

```diff
@@ -47,6 +47,26 @@ Then call `rollup` either via the [CLI](https://www.rollupjs.org/guide/en/#comma
 The plugin accepts a terser [Options](https://github.com/terser/terser#minify-options) object as input parameter,
 to modify the default behaviour.
 
+Besides, the terser options, it is also possible to pass the option `maxWorkers`,
```

Comment:
```suggestion
In addition to the `terser` options, it is also possible to provide the following options:
```

Following this line, the `maxWorkers` should be documented in the same format as the options in other plugins. e.g. https://github.com/rollup/plugins/blob/master/packages/alias/README.md#options